CONTRIBUTING.md

```diff
@@ -44,9 +44,26 @@ Other notes:
 
 ## Pull Requests
 
-To accommodate review process we suggest that PRs are categorically broken up. 
+To accommodate review process we suggest that PRs are categorically broken up.
 Ideally each PR addresses only a single issue. Additionally, as much as possible
-code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions. 
+code refactoring and cleanup should be submitted as a separate PRs from bugfixes/feature-additions.
+
+### Process for Reviewing PRs
+
+All PRs require 2 Reviews before merge. When reviewing PRs please use the following review explanations:
```

Comment:
`reviews` (no capitalization)

Reply:
Strange. I had made a suggestion and now it's gone.